README.md

```diff
@@ -89,22 +94,6 @@ return assert.isRejected(promise, Error, "optional message");
 return assert.isRejected(promise, /error message matcher/, "optional message");
 ```
 
-### Progress Callbacks
```

Comment:
at this point it might be cleaner to simply start from the current master and roll my fairly minor changes in again, rather than adding two misfire commits into the history.